crawl-ref/source/ng-setup.cc

```diff
@@ -329,15 +329,21 @@ static skill_type _setup_archaeologist_crate(item_def& crate)
 {
     item_def unrand;
     unrand_type type;
-    do {
+    do
+      {
         unrand = item_def();
         type = _archa_unrands[random2(_archa_num_unrands)];
         if (!make_item_unrandart(unrand, type))
             continue;
 
-    } while ((!you.could_wield(unrand) && !can_wear_armour(unrand, false, true)) ||  !can_generate(unrand));
+       }
+
```

Comment:
The indent on these braces isn't in-line with the `do` and the `while`. Thanks for all the other changes, they're all fine :D